internal/api/usecases/project_usecase.go

```diff
@@ -32,14 +32,18 @@ func NewProjectUsecase(projectRepo pkgrepository.ProjectRepository, metricsRepo
 }
 
 // CreateProject create and save a new project
-func (pu *ProjectUsecase) CreateProject(name string) (*pkgmodel.Project, error) {
+func (pu *ProjectUsecase) CreateProject(name string, network string) (*pkgmodel.Project, error) {
+	// this proxy can handle only network MAINNET and CARTHAGENET
+	if network != "MAINNET" && network != "CARTHAGENET" {
```

Comment:
I know this is not YAGNI but why should we not use a "not in list" like to handle it. It permits to handle other network quicker for future development. What do you think about it ?

Reply:
Agrees, create a new list of networks in config